SplitTests/Service/HttpMySegmentsFetcherTests.swift

```diff
@@ -27,7 +27,7 @@ class HttpMySegmentsFetcherTests: XCTestCase {
         restClient.isServerAvailable = false
         var isError = false
         do {
-            let _ = try fetcher.execute(userKey: "user")
+            let _ = try fetcher.execute(userKey: "user", headers: nil)
```

Comment:
It would be nice to have some asserts in E2E or integration tests to check that the `Cache-Control` header is properly set for requests due to SPLIT_UPDATE, SPLIT_KILL and MY_SEGMENTS_UPDATE (without payload) push notifications.